### PR TITLE
chore(release): 4.0.0

### PR DESCRIPTION
**Pull request by carvantes**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+
+### ⚠ BREAKING CHANGES
+
+* The Cognito `IdToken` is now used instead of the `AccessToken` to authorize requests.
+
+Multi-tenancy itself is not a breaking change, you can continue to use FHIR works on single-tenant mode 
+by not using the `enableMultiTenancy` flag. 
+  
+However, note that updating an existing (single-tenant) stack to enable multi-tenancy is a breaking change. Multi-tenant 
+deployments use a different data partitioning strategy that renders the old, single-tenant, data inaccessible.
```
Comment (rsmayda) [inline CHANGELOG.md:16]: Do we have a script for this?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fhir-works-on-aws-deployment",
-  "version": "3.1.0",
+  "version": "4.0.0",
```
Comment (Bingjiling) [inline package.json:3]: Need update to user agent string https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/serverless.yaml#L57 as well

**Review by ssvegaraju — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (carvantes) [reply CHANGELOG.md:16]: There's no script to help make this migration.

Not every existing deployment out there benefits from switching to multi-tenancy. Some customers don't have multi-tenancy requirements and others prefer multiple separate (single-tenant) stacks for different tenants.

Migrating an existing stack from single-tenant to multi-tenant can be done by exporting all data (possibly with a single bulk export operation) and then inserting it into a new multi-tenant stack.

Comment (rsmayda) [reply CHANGELOG.md:16]: Let's add this task to back log I can see this being a feature some customers would like/need

File: package.json (hunk 2 above)
Comment (carvantes) [reply package.json:3]: Done

**Review by rsmayda — APPROVED** [review]

**Pull request merged**